experimental/datacommons-js/README.md

```diff
@@ -12,11 +12,35 @@ Include datacommons.js and styles in your html `<head>...</head>` tag
 <script src="https://datacommons.org/datacommons.js"></script>
 ```
 
+For testing on autopush, replace the above script tag with
+
+```html
+<script src="https://autopush.datacommons.org/datacommons.js"></script>
+```
+
 ## Examples
 
-For a complete example, see [example.html](./example.html)
+- Web component examples: [example.html](./example.html)
+- JS examples: [example.html](./example.html)
+
+### Web Component Map
+
+Draw a choropleth map by adding the `datacommons-map` element to your HTML
+
+```html
+<datacommons-map
+  title="Population Below Poverty Level Status in Past Year in States of United States (2020)"
+  placeDcid="country/USA"
+  enclosedPlaceType="State"
+  statVarDcid="Count_Person_BelowPovertyLevelInThePast12Months"
+></datacommons-map>
+```
```

Comment:
Add brief steps on how to find place dcid and stat var dcid?

Reply:
Added